Fix build failure with gcc with -fstack-protector-all.

> $ x86_64-pc-linux-gnu-gcc -Os -pipe -march=native -frecord-gcc-switches -ggdb -D_FILE_OFFSET_BITS=64 -D_REVISION=0  -I/usr/local/include -DHAVE_SQLITE -DCONFIG_LIBNL30 -I/usr/include/libnl3  -Wno-unused-but-set-variable -Wno-array-bounds -D_FILE_OFFSET_BITS=64 -D_REVISION=0  -I/usr/local/include -DHAVE_SQLITE -DCONFIG_LIBNL30 -I/usr/include/libnl3  -Wno-unused-but-set-variable -Wno-array-bounds -Iinclude   -c -o aircrack-ptw-lib.o aircrack-ptw-lib.c
> aircrack-ptw-lib.c: Assembler messages:
> aircrack-ptw-lib.c:181: Error: symbol `.L7' is already defined

Error gets triggered on vanilla toolchain when -fstack-protector-all is used.
gcc emits labels around in start/end functions as follows (the canary check):

>    movq8(%rsp), %rcx
>    xorq%fs:40, %rcx
>    je.L7
>    call__stack_chk_fail
>.LVL5:
>.L7:
>    addq$24, %rsp
>    .cfi_def_cfa_offset 8

and gets .L7 collision.
Patch gives all .L\d names a bit nicer names to avoid collision and add readability.
diff --git a/src/aircrack-ptw-lib.c b/src/aircrack-ptw-lib.c
index 79268fb..5524053 100644
--- a/src/aircrack-ptw-lib.c
+++ b/src/aircrack-ptw-lib.c
@@ -168,10 +168,10 @@ static int rc4test_amd64_sse2(uint8_t *key, int keylen, uint8_t *iv, uint8_t *ke
 		"movdqa   (%q3), %%xmm0       \n\t"
 		"cmpl    $16, %k4             \n\t"
 		"movdqu %%xmm0, 3+"keybuf"    \n\t"
-		"jng     .L0                  \n\t"
+		"jng     .Lsmall_key1         \n\t"
 		"movdqa 16(%q3), %%xmm1       \n\t"
 		"movdqu %%xmm1,19+"keybuf"    \n\t"
-		".L0:                         \n\t"
+		".Lsmall_key1:                \n\t"
 
 		// key = keybuf
 		"lea  "keybuf", %q3           \n\t"
@@ -198,9 +198,9 @@ static int rc4test_amd64_sse2(uint8_t *key, int keylen, uint8_t *iv, uint8_t *ke
 		"movq %q4, %q8                \n\t"
 		"cmpq $16, %q8                \n\t"
 		"movq $15, %q4                \n\t"
-		"je    .L7                    \n\t"
+		"je    .Lsmall_key2           \n\t"
 		"shrq  $1, %q4                \n\t"
-		".L7:                         \n\t"
+		".Lsmall_key2:                \n\t"
 
 		// init array with key
 		".p2align 4                   \n\t"
